.github/workflows/ci.yaml

```diff
@@ -119,15 +125,23 @@ jobs:
           curl -LO https://dl.k8s.io/release/v1.27.3/bin/linux/amd64/kubectl
           chmod +x ./kubectl
           sudo mv ./kubectl /usr/local/bin/kubectl
-
+      
+      - name: Build NGINX Base image
+        if: |
+          needs.changes.outputs.baseimage == 'true'
+        run: |
+            export TAG=$(cat images/nginx-1.25/TAG)
+            cd images/nginx-1.25/rootfs && docker buildx build --platform=${{ env.PLATFORMS }} --load -t gcr.io/k8s-staging-ingress-nginx/nginx-1.25:${TAG} .
+    
       - name: Build images
         env:
           TAG: 1.0.0-dev
           ARCH: amd64
           REGISTRY: ingress-controller
         run: |
           echo "building images..."
-          make clean-image build image image-chroot
+          export TAGNGINX=$(cat images/nginx-1.25/TAG)
```

Comment:
nit: we can keep this naming consistent with line 133, but it does not affect the merging of this PR.

Reply:
no we can't :) it was a TAG here, but it conflicted with the TAG that is on line 138.

I think they have different meanings, on 133 is "what is the image of the tag going to be" while here it is "what TAG from NGINX base image I should use"